columnflow/tasks/cms/inference.py

```diff
@@ -21,13 +21,13 @@
 
 
 class CreateDatacards(
-    HistHookMixin,
+    CalibratorClassesMixin,
+    SelectorClassMixin,
+    ProducerClassesMixin,
```

Comment:
I remember that changing this order of inheritance messed up the order of how CSP init functions were being run. Is this fixed with this PR?

Reply:
Yup :+1: